blueprints/dashboards/auto-scale/config.libsonnet

```diff
@@ -0,0 +1,30 @@
+{
+  /**
+  * @param (policy.policy_name: string required) Name of the policy.
+  */
+  policy: {
+    policy_name: '__REQUIRED_FIELD__',
+  },
+  /**
+  * @param (dashboard.refresh_interval: string) Refresh interval for dashboard panels.
+  * @param (dashboard.time_from: string) From time of dashboard.
+  * @param (dashboard.time_to: string) To time of dashboard.
+  * @param (dashboard.extra_filters: map[string]string) Additional filters to pass to each query to Grafana datasource.
+  * @param (dashboard.main_dashboard_name: string) Name of the main dashboard.
+  */
+  dashboard: {
+    refresh_interval: '5s',
+    time_from: 'now-15m',
+    time_to: 'now',
+    extra_filters: {},
+    main_dashboard_name: 'Aperture Auto-scale',
```

Comment:
I think we should use `title` as name of this field instead of `main_dashboard_name`